packages/web-frontend/src/containers/MapOverlayBar/MapOverlayBar.js

```diff
@@ -221,11 +221,12 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
   const { dispatch, onSetMapOverlay, onClearMeasure } = dispatchProps;
   const { currentMapOverlayIds } = stateProps;
   const onSelectMapOverlay = mapOverlayId => {
-    onSetMapOverlay([
-      // Only two map overlays can be selected at the same time
-      ...(currentMapOverlayIds.length === 2 ? [currentMapOverlayIds[1]] : currentMapOverlayIds),
-      mapOverlayId,
-    ]);
+    const MAX_SELECTED_OVERLAYS = 2;
```

Comment:
(Detail) let's move this in the file-scope: after the imports and before other code. Symbolic constants like this one (all uppercase) usually go there, so that devs can have a look at code "configuration" at a glance